Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -231,6 +232,19 @@ public bool IsPlatformBoundaryRenderingEnabled
             set { enablePlatformBoundaryRendering = value; }
         }
 
+        [SerializeField]
+        [Tooltip("Profile for wiring up boundary visualization assets.")]
+        private MixedRealityBoundaryVisualizationProfile boundaryVisualizationProfile;
```

Comment:
Let's align the naming of this profile to reflect that it's an option.  We will have an array of profiles.

`MixedRealityBoundaryVisualizationOption[] `

Reply:
Actually, I think keeping the name is fine, but updating the variable name to be `visualiationOptions`